mLRS/CommonRx/mlrs-rx.cpp

```diff
@@ -890,12 +892,17 @@ dbg.puts(s8toBCD_s(stats.last_rssi2));*/
     //-- Store parameters
 
     if (doParamsStore) {
+        DBG_MAIN(dbg.puts("\nPARAM STORE\n"));
         sx.SetToIdle();
         sx2.SetToIdle();
         leds.SetToParamStore();
+#ifdef ESP8266
+        rxclock.disable_isr();
+#endif
         setup_store_to_EEPROM();
 #if defined(ESP8266) || defined(ESP32)
-        resetFunc(); //call reset
+        esp_setup(); //call reset
```

Comment:
this should go into setup_store_EEPROM()

try as much as you can to avoid to pollute the main code with ESP spoecific stuff